indexer/execution_client.go

```diff
@@ -205,6 +239,12 @@ func (client *ExecutionClient) processClientEvents() error {
 				return err
 			}
 			client.lastHeadRefresh = time.Now()
+
+			// update node peers
+			if err = client.updateNodePeers(ctx); err != nil {
+				return fmt.Errorf("could not get node peers for %s: %v", client.clientName, err)
+			}
```

Comment:
can you wrap this into a if block that makes the refresh less frequent?
currently it polls for every slot, which is a bit too often.
once per epoch, like for the CL peers, should be fine :)